doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+
+-----
+
+Once you've created your arrays, you can start to work with them. 
+Let's say, for example, that you've created two arrays, one called "data" and one called "ones" 
+
+.. image:: images/np_array_dataones.png
+
+You can add the arrays together with the plus sign.
+
+::
+
+  data + ones
+
+.. image:: images/np_data_plus_ones.png
+
+You can, of course, do more than just addition!
+
+::
+
+  data - ones
+  data * data
+  data / data
+
+.. image:: images/np_sub_mult_divide.png
+
+Basic operations are simple with NumPy. If you want to find the sum of the elements in an array, you'd use ``sum()``. This works for 1D arrays, 2D arrays, and arrays in higher dimensions.
+
+**Input:**
+
+::
+
+  a = np.array([1, 2, 3, 4])
+
+  # Add all of the elements in the array
+  a.sum()
+
+**Output:**
+
+::
+
+  10
+
+To add the rows or the columns in a 2D array, you would specify the axis.
+
+If you start with this array:
+
+**Input:**
+
+::
+
+  b = np.array([[1, 1], [2, 2]])
+
+You can sum the rows with:
+
+**Input:**
+
+::
+  
+  b.sum(axis=0)
+
+**Output:**
+
+::
+
+  array([3, 3])
+
+You can sum the columns with:
+
+**Input:**
+
+::
+
+  b.sum(axis=1)
+
+**Output:**
+
+::
+
+  array([2, 4])
+
+`Learn more about basic operations here <https://docs.scipy.org/doc/numpy-1.17.0/user/quickstart.html#basic-operations>`_.
+
+
+Broadcasting
+------------
+
+There are times when you might want to carry out an operation between an array and a single number (also called *an operation between a vector and a scalar*) or between arrays of two different sizes. For example, your array (we'll call it "data") might contain information about distance in miles but you want to convert the information to kilometers. You can perform this operation with: 
+
+::
+
+  data * 1.6
+
+.. image:: images/np_multiply_broadcasting.png
+
+NumPy understands that the multiplication should happen with each cell. That concept is called **broadcasting**. Broadcasting is a mechanism that allows NumPy to perform operations on arrays of different shapes. The dimensions of your array must be compatible, for example, when the dimensions of both arrays are equal or when one of them is 1. If the dimensions are not compatible, you will get a value error. 
+
+`Learn more about broadcasting here <https://docs.scipy.org/doc/numpy-1.17.0/user/basics.broadcasting.html>`_.
+
+
+More useful array operations
+-----------------------------------
+
+
+::
+
+  Maximum, minimum, sum, mean, product, standard deviation, and more
+
+NumPy also performs aggregation functions. In addition to ``min``,  ``max``, and ``sum``, you can easily run ``mean`` to get the average, ``prod`` to get the result of multiplying the elements together, ``std`` to get the standard deviation, and more.
+
+::
+
+  data.max()
+  data.min()
+  data.sum()
+
+.. image:: images/np_aggregation.png
+
+Let's start with this array, called "A"
+
+::
+
+ [[0.45053314 0.17296777 0.34376245 0.5510652]
```

Comment:
indent only one space, should be 2 spaces to render I think. Also, easier for the user to add `A =` so they can copy-paste probably

Reply:
while we're at it: can you use a lower-case name (`a` or `x`)? better teach the right habits:)

Reply:
Got it!